src/faction_camp.cpp

```diff
@@ -1909,25 +1909,29 @@ static std::pair<size_t, std::string> farm_action( const tripoint &omt_tgt, farm
                 }
                 break;
             case farm_ops::harvest:
-                if( farm_map.furn( pos ) == f_plant_harvest && !farm_map.i_at( pos ).empty() ) {
-                    const item &seed = farm_map.i_at( pos ).front();
-                    if( farm_valid_seed( seed ) ) {
+                if( farm_map.furn( pos ) == f_plant_harvest ) {
+                    // Can't use item_stack::only_item() since there might be fertilizer
```

Comment:
It should be checking for the `GROWTH_HARVEST` flag, but it looks as though this whole thing is largely a duplicate of `iexamine::harvest`, and is out of date with the current state of what `iexamine::harvest` does.

Reply:
This function led me to believe that fertilizer stays on the tile.  It could also easily be outdated though.
https://github.com/CleverRaven/Cataclysm-DDA/blob/4d34142d4cd0666b48bfd8ffce67a22af63608da/src/iexamine.cpp#L2066-L2082

Reply:
You're right.  I did read `iexamine::fertilize_plant` but I missed that line.